app/views/content/service_privacy_notice.md

```diff
@@ -57,14 +57,14 @@ If you process teacher training applications, we may need to collect information
 - where you work
 - what your role is
 
-#### The data we collect if you’re a referee
+#### The data we collect if you give a reference
 
-If you’re a referee, we collect your personal details from a candidate who selected you as a referee. They told us your:
+If a candidate says that you can give them a reference, they’ll tell us:
 
-- name
-- email address
-- relationship to the candidate
-- reference type (for example, academic or personal)
+- your name
+- your email address
+- how you know them and how long you’ve known them
+- the type of reference you can give (for example, academic or professional)</li>
```

Comment:
think we can do without `</li>`

Reply:
Oops sorry. That's what happens when I try to mirror changes to an HTML file in a markdown file...